src/OrbitGl/App.cpp

```diff
@@ -1328,9 +1328,12 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> OrbitApp::LoadCaptureFro
 
 orbit_base::Future<ErrorMessageOr<void>> OrbitApp::MoveCaptureFile(
     const std::filesystem::path& src, const std::filesystem::path& dest) {
+  std::optional<absl::Duration> capture_length =
+      capture_file_info_manager_.GetCaptureLengthByPath(src);
```

Comment:
You could also use `GetCaptureTime()` here? Is there a benefit of using capture_file_info_manager that I am not seeing?

Reply:
I thought about doing that before. But if we use `GetCaptureTime()` here, then that's kind of assuming the capture file with the path `src` is now opening in `OrbitApp`.

If we change the method `OrbitApp::MoveCaptureFile(const std::filesystem::path& src, const std::filesystem::path& dest)` to  `OrbitApp::MoveCaptureFile(const std::filesystem::path& dest)`, then I would like to use `GetCaptureTime()`. 

But the benefit for using `OrbitApp::MoveCaptureFile(const std::filesystem::path& src, const std::filesystem::path& dest)` and `GetCaptureLengthByPath(src)` is that we can use this to move whatever capture file.

Reply:
sgtm!